README.md

```diff
@@ -213,32 +223,27 @@ Finally, we use `host` `network_mode`, this is because our server code refers to
 
 </details>
 
-## Start
-
-Make sure you had built the image via `bash build_image.sh`
-Start all services at once with the following command, however it may clog up the terminal window as they all run in the same terminal so you may run some services in detached mode.
-
-`docker compose up`
+## Quick Start
 
-To start multiple containers on a machine, use the following command
+We require our Params FTP Server to be running before starting the prover node. The prover node must copy the parameters from the FTP server to it's own volume to operate correctly.
 
-`docker compose -p <node> up` where `node` is the unique name of the container/project you would like to start.
+### Params FTP Server
 
-Ensure the docker compose file has GPU's specified for each container.
+Start the FTP server with `docker compose up params-ftp -f ftp-docker-compose.yml`.
```

Comment:
This command run fail:
yymone@ps:~/MultiProveNode/proverDockerNode2/prover-node-docker$ docker compose up params-ftp -f ftp-docker-compose.yml
unknown shorthand flag: 'f' in -f


I change to " docker compose  -f ftp-docker-compose.yml up params-ftp" and success, so need change here?